pkgs/tools/networking/i2p/default.nix

```diff
@@ -1,60 +1,98 @@
 { lib
 , stdenv
-, ps
-, coreutils
-, fetchurl
+, fetchzip
 , jdk
-, jre
 , ant
 , gettext
 , which
+, dbip-country-lite
 , java-service-wrapper
+, makeWrapper
+, gmp
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "i2p";
   version = "2.4.0";
 
-  src = fetchurl {
-    urls = map (mirror: "${mirror}/${finalAttrs.version}/i2psource_${finalAttrs.version}.tar.bz2") [
-      "https://download.i2p2.de/releases"
-      "https://files.i2p-projekt.de"
-      "https://download.i2p2.no/releases"
+  src = fetchzip {
+    urls = map (mirror: "${mirror}${finalAttrs.version}/i2psource_${finalAttrs.version}.tar.bz2") [
+      "https://github.com/i2p/i2p.i2p/releases/download/i2p-"
+      "https://download.i2p2.de/releases/"
+      "https://files.i2p-projekt.de/"
+      "https://download.i2p2.no/releases/"
     ];
-    sha256 = "sha256-MO+K/K0P/6/ZTTCsMH+GtaazGOLB9EoCMAWEGh/NB3w=";
+    hash = "sha256-JfPjjNVtXg5LCpg+k2973WxFmsChOyUMPGb8v4J8fRw=";
+    # Remove unused binaries
+    postFetch = "rm -r $out/installer/lib";
```

Comment:
This makes it harder to reproduce the download entry. Is there another reason than cleanup to do this? If not doing this in postUnpack would be nicer.

Reply:
I want to make the src tarball smaller. Since it's not preferred I removed this.